.circleci/install.sh

```diff
@@ -0,0 +1,41 @@
+#! /bin/bash -ex
+# -e means:  Exit immediately if a command exits with a non-zero status
+# -x means:  Print commands and their arguments as they are executed.
+
+# Install some required system packages
+sudo apt-get update
+sudo apt-get install libegl1-mesa
+
+# Install Miniconda
+wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
+bash miniconda.sh -b -p $HOME/miniconda
+export PATH="$HOME/miniconda/bin:$PATH"
+
+# Install required Python version
+conda install -y python=$PYTHON_VERSION
```

Comment:
You're installing everything in conda's base or root environment.

That's bad. Instead, please create a clean env (ci-helpers calls it `test`) and install everything there.